Runtime/src/index.ts

```diff
@@ -117,25 +122,33 @@ class SwiftRuntimeHeap {
     }
 }
 
+/// Memory lifetime of closures in Swift are managed by Swift side
+class SwiftClosureHeap {
+    private functionRegistry: FinalizationRegistry<number>;
+    private exports: SwiftRuntimeExportedFunctions
+
+    constructor(exports: SwiftRuntimeExportedFunctions) {
+        this.exports = exports
+        this.functionRegistry = new FinalizationRegistry((id) => {
+            this.exports.swjs_free_host_function(id);
```

Comment:
Why not reference `exports` directly, removing the need for the `exports` property?
```suggestion
            exports.swjs_free_host_function(id);
```